mobile/native/src/trade/order/handler.rs

```diff
@@ -35,6 +38,7 @@ pub async fn submit_order(order: Order) -> Result<()> {
     }
 
     update_order_state_in_db_and_ui(order.id, OrderState::Open)?;
+    position::handler::update_position_after_order_submitted(order)?;
```

Comment:
🤔 Since our close button state is dependent on the Closing state of the position, isn't it risky to wait so long to set the state of the position to `Closing`. e.g. It sometimes takes quite some time for the `post_order` http call to return. In that that time the user could click close position again? leading to other potential issues?